### ISSUE TITLE
Can't get value of csharp:InterpolatedStringText node

### ISSUE THREAD

**Issue by erizocosmico** [bblfsh/csharp-driver#21]
I have this code:

```csharp
namespace SomeNamespace
{
    public static class SomeClass
    {
        public static String GetFromFoo(field String)
        {
            return " SELECT " + param + " FROM foo";
        }
      
      	public static String GetFromFooWithInterpolation(field String)
        {
            return $"SELECT {param} FROM foo";
        }
    }
}
```

This query returns the `" SELECT "` node:

```
//csharp:BinaryExpression_AddExpression/Left/uast:String[string-length(@Value) > 0]
```

However, this does not return the `$"SELECT "` node:

```
//csharp:InterpolatedStringExpression/Contents/csharp:InterpolatedStringText[1]/TextToken[string-length(@Value) > 0]
```

Even though I can see inspecting that it _does_ have a `Value` attribute.

<img width="544" alt="screenshot 2019-02-05 at 15 48 13" src="https://user-images.githubusercontent.com/1312023/52281042-80233880-295d-11e9-81de-c345e1dab0b7.png">

So far, it only happens with that node.

**Comment by dennwc** [comment]
@erizocosmico Note that you are writing it with `InterpolatedStringText`, while the screenshot shows `InterpolatedStringTextToken` node. Have you missed the `Token` part?

**Comment by dennwc** [comment]
You have the following structure:
```
InterpolatedStringExpression - Contents -> [ InterpolatedStringText ] - TextToken -> InterpolatedStringTextToken - Value -> "some string"
```
So you are missing the `InterpolatedStringTextToken` indirection.

This should work, I think:
```
//csharp:InterpolatedStringExpression/Contents/csharp:InterpolatedStringText[1]/TextToken/csharp:InterpolatedStringTextToken[string-length(@Value) > 0]
```

Note that #23 will simplify it.

**Comment by erizocosmico** [comment]
Ok, got it. Thanks!

**Comment by dennwc** [comment]
@erizocosmico The new version was released: [v1.2.0](https://github.com/bblfsh/csharp-driver/releases/tag/v1.2.0). It will require to change the query, but it will be shorter.
